app.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Les meves ajudes",
+  "description": "",
+  "env": {
+    "APP_NAME": {
+      "value": "Les meves ajudes"
+    },
+    "NODE_ENV": {
+      "value": "production"
+    }
+  },
+  "image": "heroku/ruby",
```

Comment:
Why not `"image": "heroku/nodejs"` ?

Reply:
Definitely a mistake. `app.json` only gets used when creating a new app, so I didn't catch this. Thanks!